web/packages/teleport/src/features.tsx

```diff
@@ -392,6 +396,35 @@ export class FeatureDiscover implements TeleportFeature {
   }
 }
 
+export class FeatureIntegrations implements TeleportFeature {
+  category = NavigationCategory.Management;
+  section = ManagementSection.Access;
+
+  hasAccess(flags: FeatureFlags) {
+    return flags.integrations;
+  }
+
+  route = {
+    title: 'Manage Integrations',
+    path: cfg.routes.integrations,
+    exact: true,
+    component: () => <Integrations />,
+  };
+
+  navigationItem = {
+    title: 'Integrations',
+    icon: <IntegrationsIcon />,
+    exact: true,
+    getLink() {
+      return cfg.routes.integrations;
+    },
+  };
+
+  getRoute() {
+    return this.route;
+  }
```

Comment:
What's this for?

Reply:
it's used like this here in the enterprise PR: https://github.com/gravitational/teleport.e/pull/1081/files#diff-3eb6ba1cbad5cfeecd00a9bdbe2e8942fbcabcfbda23e3f59cd72b7ba1220434R211 (below), i wanted to use the same route fields defined in OS, but just change the component part:

```ts
class FeatureIntegrations extends OSS.FeatureIntegrations {
  route = {
    ...super.getRoute(),
    // Enterprise version includes the enterprise only
    //  "plugin" resource along with the base
    // "integration" resource.
    component: () => <Integrations />,
  };

  hasAccess(flags: FeatureFlags) {
    return flags.plugins || flags.integrations;
  }
}
```